core/finality-grandpa/Cargo.toml

```diff
@@ -23,13 +23,18 @@ network = { package = "substrate-network", path = "../network" }
 service = { package = "substrate-service", path = "../service", optional = true }
 srml-finality-tracker = { path = "../../srml/finality-tracker" }
 fg_primitives = { package = "substrate-finality-grandpa-primitives", path = "primitives" }
-grandpa = { package = "finality-grandpa", version = "0.7.2", features = ["derive-codec"] }
+grandpa = { package = "finality-grandpa", git = "https://github.com/paritytech/finality-grandpa/", branch = "mod-equivocation-reports", features = ["derive-codec"] }
```

Comment:
IMO we should block this on getting upstream features merged and a new version pushed out